ISSUE_TEMPLATE/generic_issue.yml

```diff
@@ -0,0 +1,47 @@
+name: Generic Container Image Issue
+description: File a generic issue here
+title: "[Issue]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        <!---
+        Thank you for taking the time to report this issue to us. We appreciate the feedback and use reports such as this one to guide our efforts at improving our products. That being said, this bug tracking system is not a mechanism for requesting support, and we are not able to guarantee the timeliness or suitability of a resolution.
+
+        If this issue is critical or in any way time sensitive, please raise a ticket through the regular Red Hat support channels to ensure it receives the proper attention and prioritization to assure a timely resolution.
+
+        For information on how to contact the Red Hat production support team, please visit:
+            https://access.redhat.com/support
+        -->
+
+  - type: dropdown
+    attributes:
+      label: Container platform
+      description: Is your issue related to running containers on OCP or by podman/docker?
+      options:
+        - OCP
```

Comment:
OCP == OpenShift Container Platform. We can define only OpenShift 3 or OpenShift 4. But maybe nitpicks.